packages/angular/lib/src/errorhandler.ts

```diff
@@ -26,7 +26,7 @@ export interface ErrorHandlerOptions {
 class SentryErrorHandler implements AngularErrorHandler {
   protected readonly _options: ErrorHandlerOptions;
 
-  public constructor(options?: ErrorHandlerOptions) {
+  public constructor(@Inject('errorHandlerOptions') options?: ErrorHandlerOptions) {
```

Comment:
This `@Inject(...)` Decorator is required by the Angular compiler as the parameter has to be passed to the constructor for DI

Note to myself: check if this is actually true w/ Angular 10